dev/com.ibm.ws.jpa_spec10_injection_fat/fat/src/com/ibm/ws/jpa/injection/dfi/JPA10Injection_DFI_NoInheritance_EJB.java

```diff
@@ -187,6 +191,13 @@ public boolean include(ArchivePath arg0) {
         appRecord.setLocation(applicationName + ".ear");
         appRecord.setName(applicationName);
 
+        if (FATSuite.repeatPhase != null && FATSuite.repeatPhase.contains("hibernate")) {
```

Comment:
we should move this code block to before the server starts, so we don't need to deal with a dynamic config change + app restart right away

Reply:
The test application is installed after a tool application (that manages the database contents) is installed and invoked.  This tool is responsible for creating the tables that the JPA application expects to already exist on the database.  Thus, this needs to remain.